pkg/ride/evaluator/evaluate/evaluate.go

```diff
@@ -22,3 +24,22 @@ func Eval(e ast.Expr, s ast.Scope) (bool, error) {
 
 	return b.Value, nil
 }
+
+func Verify(scheme byte, state types.SmartState, script *ast.Script, transaction proto.Transaction) (bool, error) {
```

Comment:
`scheme` can be retrieved using `state.BlockchainSettings().AddressSchemeCharacter`, maybe we shouldn't add extra argument here.

Reply:
Still applies.

Reply:
Still applies.